e2e/service/trust_test.go

```diff
@@ -0,0 +1,9 @@
+package service
+
+import (
+	"testing"
+)
+
+func TestServiceCreateWithContentTrust(t *testing.T) {}
+
+func TestServiceUpdateWithContentTrust(t *testing.T) {}
```

Comment:
remove for now and open an issue for these?

Reply:
huh :sweat: